CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - `standard` formatting for JavaScript [#736]
 - Interaction for visual regression tests [#748]
+- Fixed the youtube failing visual regression test [#761]
```

Comment:
This needs to go under a `### Fixed` section. See "What makes a good change log?" of http://keepachangelog.com/